server/app/controllers/users.js

```diff
@@ -0,0 +1,36 @@
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import db from '../db/dbconnect';
+
+
+/**  create a User
+ * @param {string} email
+ * @param {string} password
+ * @param {string} user
+ * @param {string} address
+ * @return {string} token
+ * @public
+*/
+const createUser = (email, password, user, address) => {
+  const hashedPassword = bcrypt.hashSync(password, 8);
```

Comment:
You are not calling this function appropriately according to the [documentation](https://www.pivotaltracker.com/n/projects/2196560).

Reply:
the documentation you sent is for pivotal tracker